gen.mk

```diff
@@ -0,0 +1,26 @@
+# Helper for generating image repository files
+
+ifndef common_dir
+    common_dir = common
+endif
+
+generator = $(common_dir)/generate.sh
+DISTGEN_BIN ?= /usr/bin/dg
```

Comment:
Could you somewhere fail, or warn at least if user does not have this binary? Where does one install it from?

Reply:
dnf install distgen :D